documentation/htdocs/config.txt

```diff
@@ -1,4 +1,4 @@
-# Uncrustify-0.66
+# Uncrustify-0.66.1
```

Comment:
if they've changed the versioning scheme upstream it could break some things on our side (version parsing).